### PR TITLE
fix(arithmetic): Autocomplete breaks on invalid terms

### PR DESCRIPTION
**Pull request by wmak**
- When an arrow key is hit when the current term doesn't exist
  autocomplete errors
- Fixes JAVASCRIPT-24ZG

### CODE REVIEW

File: tests/js/spec/views/eventsV2/table/arithmeticInput.spec.tsx [hunk 1]
```diff
@@ -180,4 +180,13 @@ describe('ArithmeticInput', function () {
...
   });
+
+  it('handles autocomplete on invalid term', function () {
+    const input = wrapper.find('input');
+    input.simulate('focus');
+
+    const value = 'foo + bar';
+    input.simulate('change', {target: {value}});
+    input.simulate('keydown', {key: 'ArrowDown'});
+  });
```
Comment (dashed) [inline tests/js/spec/views/eventsV2/table/arithmeticInput.spec.tsx:191]: This test seems to be missing an assertion.

Comment (wmak) [reply tests/js/spec/views/eventsV2/table/arithmeticInput.spec.tsx:191]: Not sure what to assert here, I'm only trying to validate that this sequence of actions doesn't raise an error

Comment (dashed) [reply tests/js/spec/views/eventsV2/table/arithmeticInput.spec.tsx:191]: You can make an assertion of what the UI should expect to look like for this edge case.

Comment (wmak) [reply tests/js/spec/views/eventsV2/table/arithmeticInput.spec.tsx:194]: @dashed added one in eff7932

**Review by dashed — APPROVED** [review]

**Review by Zylphrex — APPROVED** [review]
Nice! I was wondering what that error was.
